docs/source/algorithms/IntegratePeaksSkew-v1.rst

```diff
@@ -0,0 +1,160 @@
+determined to find the detectors contributing to the peak; data in these detectors are then focused (and a
+background subtracted) before applying the same method to identify the bins that contribute to the peak in the focused
+spectrum. The full TOF extent of the peak is determined by maximising :math:`I/\sigma` (similar to integration methods
+for step-scans with monochromatic x-rays [3]_) starting from a seed of bins identified using the skew method.
+The user can then optionally use the optimal TOF extent to repeat the procedure to find the detectors contributing to
+the peak.
+
+Technically this algorithm does not use the a seed-skew algorithm that grows a seed, rather non-background points found
+by minimising the skew of the background, subsequently nearest-neighbour connected regions are found in the
+non-background points and a single region is assigned to the peak (by proximity to the expected peak position).
+
+Note this algorithm applies the Lorentz correction to the integrated intensity.
+
+The algorithm proceeds as follows:
+
+1.  Calculates an initial TOF window using the input parameter ``FractionalTOFWindow`` or (if ``FractionalTOFWindow = 0``)
+    evaluating a resolution function of the form
+
+        .. math::
+
+            \frac{dTOF}{TOF} = \sqrt{(\cot(\theta)d\theta)^2 + \left(\frac{dt_0}{t_0}\right)^2 + \left(\frac{dL}{L}\right)^2}
+
+    where :math:`dTOF` refers to the integration window size rather than the FWHM of a peak, the parameter
+    ``BackscatteringTOFResolution`` is the fractional window size (:math:`dTOF/TOF`) of a peak at backscattering
+    (equivalent to :math:`\sqrt{(\frac{dt_0}{t_0})^2 + (\frac{dL}{L})^2}`) and ``ThetaWidth`` :math:`= d\theta`
+    (which is adjusted to reproduce the window size required for peaks at larger scattering angles).
+
+2. Integrate the data over the range :math:`TOF_{peak} \pm 0.5*dTOF`
+
+3. Find the detectors contributing to the background by minimising the skew of the integrated intensity of detectors in
+   a square window of side length ``2*NPixels + 1``.
+
+4. Take the peak mask (detectors which contribute to the peak) to be the contiguous region of nearest-neighbour
+   connected detectors not contributing to the background that contains the peak detector ID
+   (or if ``UseNearest=True`` then it will be the nearest region).
+
+5. Focuses the data (in TOF) for detectors assigned to the peak subtracting the spectra from a shell of
+   adjacent background pixels. This has the added benefit of roughly subtracting powder lines (if the detectors are
+   close enough for powder lines to roughly line up in TOF when averaged over the mask/shell).
+
+6. Find the bins in the focused spectrum contributing to the peak by minimising the skew in the TOF window.
+
+7. Increase the TOF window until the ratio of :math:`I/\sigma` reaches a maximum.
+
+8. If ``OptimiseMask=True`` then steps 4-7 will be repeated using the optimal TOF window to find the detectors
```

Comment:
Is it steps 3-7 that get repeated? Looks like `find_peak_mask` gets called again which includes the skew minimisation